README.md

```diff
@@ -28,6 +28,8 @@ jobs:
         run: echo "Built release ID ${{ steps.build.outputs.release_id }}"
 ```
 
+`BALENA_API_KEY` may be used instead of `BALENA_TOKEN`. This is the recommended secret name.
```

Comment:
Only `balena_token` is accepted: https://github.com/balena-io/deploy-to-balena-action/blob/v0.17.0/action.yml#L7-L9

Reply:
Thanks @Page-. Lemme take another look

Reply:
Gave it another shot. I'm getting more familiar with the yaml file and the flow of data. 2nd time's a charm @Page- ?

Reply:
Oops, missed 1 minor thing in main.ts. 1 sec